.env.dev

```diff
@@ -19,3 +19,4 @@ TRACKING_STRATEGY=logging
 HACK_LOCAL_ROOT_PARENT=/tmp
 WEBAPP_URL=http://localhost:8000/
 API_URL=http://localhost:8001/api/v1/
+INTERNAL_API_URL=http://airbyte-server:8001/api/
```

Comment:
This line looks incorrect.

Reply:
ah yes, this should be `INTERNAL_API_HOST`